mindmeld/models/helpers.py

```diff
@@ -191,6 +200,7 @@ def register_model(model_type, model_class):
         model_type (str): The model type as specified in model configs
         model_class (class): The model to register
     """
+    # TODO: deprecate MODEL_MAP var in in lieu of ModelFactory
```

Comment:
Is this for the current PR or a later change?

Reply:
for a later PR